src/Sulu/Bundle/AdminBundle/Command/ValidateBuildCommand.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Command;
+
+use Symfony\Component\Asset\Packages;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Webmozart\Assert\Assert;
+
+#[AsCommand(
+    name: 'sulu:admin:validate-build',
+    description: 'Downloads the current admin application build from the sulu/skeleton repository.',
+)]
+class ValidateBuildCommand extends Command
+{
+    public function __construct(
+        private Packages $assets,
+        private string $publicDir,
+        private string $suluVersion,
+    ) {
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $adminBuildVersion = $this->getJavascriptBuildVersion($io);
+
+        if ($adminBuildVersion !== $this->suluVersion) {
+            $io->error(<<<TEXT
+                Version missmatch:
+                Admin bundle version: {$adminBuildVersion}
+                Sulu Backend version: {$this->suluVersion}
+                TEXT);
+
+            return self::FAILURE;
+        } else {
+            $io->success('JS Build is up to date.');
+        }
+
+        $input->setInteractive(false);
+
+        return $this->getApplication()->doRun(new ArrayInput([
+            'command' => 'lint:container',
+        ]), $output);
```

Comment:
This has nothing todo with validating the admin build and should be removed

```suggestion
        return 0;
```

Reply:
In the issue you said something about running composer lint. I don't know what this was about.

Reply:
in the skeleton we require then a PR which adds a new `lint-admin-build` command which is added to the. `composer lint` list here: https://github.com/sulu/skeleton/blob/590f5b77073a55a1bc6a31fdf85304cc8648a71c/composer.json#L137-L149

Reply:
Ah okay, so nothing to worry here but another PR for the skeleton when this is merged?